pkg/test/kube/accessor.go

```diff
@@ -336,6 +336,15 @@ func (a *Accessor) ValidatingWebhookConfigurationExists(name string) bool {
 	return err == nil
 }
 
+// GetValidatingWebhookConfigurationGeneration returns the current generation number (useful for detecting updates)
+func (a *Accessor) GetValidatingWebhookConfigurationGeneration(name string) (int64, error) {
```

Comment:
This is extremely specific to Istio internals. This should simply be a Getter implementation for ValidatingWebhookConfiguration. Istio specific parts can be supplied as parameters

Reply:
I'm not sure what you mean by "istio specific parts" but I was able to just change this method to GetValidatingWebhookConfiguration and extract the generation in the test. Please let me know if you had something else in mind.